src/state/contexts/GlobalState.tsx

```diff
@@ -1,13 +1,14 @@
+import { JWKInterface } from 'arweave/node/lib/wallet';
 import React, { Dispatch, createContext, useContext, useReducer } from 'react';
 
-import type { ArNSContractState, ArweaveWalletConnector } from '../../types';
+import type { ArNSContractState } from '../../types';
 import type { Action } from '../reducers/GlobalReducer';
 
 export type GlobalState = {
   arnsSourceContract: ArNSContractState;
   gateway: string;
   walletAddress?: string;
-  wallet?: ArweaveWalletConnector;
+  jwk?: JWKInterface;
```

Comment:
don't revert this change, make sure you pull most recent version of develop via `git merge origin/develop`

Reply:
wierd.... merged... file not updating... must have merged then overwrote it....

Reply:
anyway, added it

Reply:
we don't want jwk at all, we'll use the `ArweaveWalletConnector` or something different here